dev-docs/bidders/theAdx.md

```diff
@@ -1,11 +1,11 @@
 ---
 layout: bidder
-title: theadx
+title: TheAdx
 description: Prebid TheAdx Bidder Adapter
 pbs: true
 pbs_app_supported: true
 pbjs: true
-biddercode: theadx
+biddercode: theAdx
```

Comment:
```suggestion
biddercode: theadx
```

The bidder code must match the one provided in the bid adapter

https://github.com/prebid/Prebid.js/blob/69d79062781c38d8fafcac189f807e99c4b28f65/modules/theAdxBidAdapter.js#L21

Reply:
@mustafakemal16 can you please update the PR accordingly 😊

Reply:
Hi @muuki88 ,  when selecting a bidder from the Prebid download page, if the bidder class name and bidder code don't match, the download cannot be completed. For example, trying to download the theadx bidder from https://docs.prebid.org/download.html results in an error. Therefore, the prefix in theAdxBidAdapter.js must match the bidder code. If you want to download theAdxBidAdapter, the bidder code should be 'theAdx'. Please correct me if I'm wrong.

Reply:
The problem here is that the biddercode is lowercase `theadx` while the filename is mixed case `theAdx`. Not ideal, but not the first time this has happened. I'll fix this with the filename header.